projects/common/de10nano/de10nano_system_assign.tcl

```diff
@@ -0,0 +1,704 @@
+# de10nano
+# clocks (V11, Y13, E11 - PL 50MHz)
+# clocks (E20, D20 - HPS 25MHz)
+
+set_location_assignment PIN_V11 -to sys_clk
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to sys_clk
+
+# Switches
+
+set_location_assignment PIN_Y24  -to gpio_bd_o[0]
+set_location_assignment PIN_W24  -to gpio_bd_o[1]
+set_location_assignment PIN_W21  -to gpio_bd_o[2]
+set_location_assignment PIN_W20  -to gpio_bd_o[3]
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to gpio_bd_o[0]
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to gpio_bd_o[1]
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to gpio_bd_o[2]
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to gpio_bd_o[3]
+
+# LEDs
+
+set_location_assignment PIN_W15   -to gpio_bd_i[0]
+set_location_assignment PIN_AA24  -to gpio_bd_i[1]
+set_location_assignment PIN_V16   -to gpio_bd_i[2]
+set_location_assignment PIN_V15   -to gpio_bd_i[3]
+set_location_assignment PIN_AF26  -to gpio_bd_i[4]
+set_location_assignment PIN_AE26  -to gpio_bd_i[5]
+set_location_assignment PIN_Y16   -to gpio_bd_i[6]
+set_location_assignment PIN_AA23  -to gpio_bd_i[7]
```

Comment:
Shouldn't the LEDs be outputs from FPGA point of view ?

Reply:
Fixed it. Thank you.